includes/results.inc

```diff
@@ -1099,6 +1099,8 @@ class IslandoraSolrFacets {
 
     // Show more link.
     if (count($buckets) > $soft_limit) {
+      // Create hook_islandora_solr_facet_variables_alter().
+      drupal_alter('islandora_solr_facet_variables', $buckets);
```

Comment:
This only runs if you have more facet values then the soft limit (ie. you need to add the "show more" link). You probably want to move this above the 
```
    // Show more link.
    if (count($buckets) > $soft_limit) {
```
and alter the hook definition in the api to assume an array but it could be 0 length.

That or add a duplicate call below between lines 1118 and 1119.